volume/volume.go

```diff
@@ -205,6 +205,8 @@ type ProtoDriver interface {
 	Status() [][2]string
 	// Shutdown and cleanup.
 	Shutdown()
+	// DU specified volume and potentially the subfolder if provided.
+	Catalog(volumeid, subfolder string) (api.CatalogResponse, error)
```

Comment:
I think this API needs to be a little clearer. For example: `volumeid, subfolder string, depth int`, That way it only goes so deep into the directory.

Reply:
done in new pr :)